src/components/Text/Text.styles.ts

```diff
@@ -0,0 +1,49 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { TextProps, TextSize } from './Text';
+import { ellipsisStyle, fontWeightBold, fontSizes, lineHeights, textColors, Sizes } from '../../util/styles/fonts';
+import { IRawStyle } from '@uifabric/styling';
+
+const heights: Sizes = {
```

Comment:
why are heights and verticalAligns duplicated here instead of using the exported values from fonts.ts directly?